### PR TITLE
Ndss 2021

### PR DESCRIPTION
**Pull request by luxroot**
Added WINNIE, PGFUZZ

### CODE REVIEW

**Comment by sangkilc** [comment]
Conflict

**Comment by sangkilc** [comment]
AFL-HIER was merged to the master

**Comment by luxroot** [comment]
I rebased it with master.

File: data/fuzzers.json [hunk 1]
```diff
@@ -3537,6 +3591,34 @@
...
   },
+  {
+    "name": "WINNIE",
+    "year": 2021,
+    "author": [
+      "Jinho Jung",
+      "Stephen Tong",
+      "Hong Hu",
+      "Jungwoo Lim",
+      "Yonghwi Hin",
```
Comment (sangkilc) [inline data/fuzzers.json:3602]: Typo in the name

Comment (luxroot) [reply data/fuzzers.json:3602]: Sorry I'll fix it

File: data/fuzzers.json [hunk 2]
```diff
@@ -3537,6 +3561,34 @@
...
+      "https://huhong789.github.io/papers/winnie-ndss2021.pdf"
+    ],
+    "targets": [
+      "File"
+    ],
+    "references": [
+      "AFL"
+    ],
+    "keywords": [
+      "Windows"
```
Comment (sangkilc) [inline data/fuzzers.json:3618]: Add GUI maybe?

Comment (luxroot) [reply data/fuzzers.json:3618]: That makes sense. I was wondering whether to put it or not. I'll add it right away.

**Pull request merged**